### PR TITLE
Fix pagination - page limit and sort by issue

### PR DESCRIPTION
**Pull request by djtimca**
Made pageLimit easily changeable as a parameter (ie. &pageLimit=100).

Append sort by and page limit when moving between pages.

### CODE REVIEW

File: components/LocalList.php [hunk 1]
```diff
@@ -80,6 +80,11 @@ protected function loadList()
 
         $this->mapIntoObjects($list);
 
+        if ($sortBy) {
+            $list->appends('sort_by', $sortBy);
+            $list->appends('pageLimit', $this->param('pageLimit'));
+        }
+
```
Comment (sampoyigi) [inline components/LocalList.php:87]: Do we really need this? Why was it added?

Comment (djtimca) [reply components/LocalList.php:87]: If you have the list sorted and there are more locations than the default page limit, clicking to the next page eliminates the sorting on the page. This change keeps the sorting when you go to page 2 and allows you to set the max # of locations on a single page.

Comment (sampoyigi) [reply components/LocalList.php]: Thank you! I should have checked the Laravel docs instead, first time seeing the appends method. 

I've pushed some changes let me know if it's okay to merge.

Comment (djtimca) [reply components/LocalList.php]: Looks good to me.

**Pull request merged**